### PR TITLE
Commit 13 - add faq page in struts

### PR DESCRIPTION
**Pull request by Trjegul84**
No changes related to non existen file src/main/resources/ApplicationResources_es.native
Changes already done in previous PR:

-  src/main/resources/ApplicationResources_es.properties
- src/main/webapp/WEB-INF/pages/faq.ftl

New additions only in  src/main/resources/struts.xml and src/main/webapp/WEB-INF/pages/inc/menu.ftl 
NOTE:
FAQ page content is not yet visible with this commit

### CODE REVIEW

File: src/main/webapp/WEB-INF/pages/inc/menu.ftl [hunk 1]
```diff
@@ -61,8 +61,7 @@
                         <li class="nav-item">
                             <a class="nav-link [#if currentMenu=='admin']active[/#if]" href="${baseURL}/admin/">[@s.text name="menu.admin"/]</a>
                         </li>
-                    [/#if]
-                    [#if !adminRights && !managerRights]
+                    [#else]
```
Comment (alegnaaived) [inline src/main/webapp/WEB-INF/pages/inc/menu.ftl:64]: In this commit was removed target="_blank" in line 85

**Review by alegnaaived — CHANGES_REQUESTED** [review]
It is necessary to check missing file to modify:

- src/main/webapp/WEB-INF/pages/faq.ftl
- ssrc/main/resources/ApplicationResources_es.properties

File: src/main/webapp/WEB-INF/pages/inc/menu.ftl (hunk 1 above)
Comment (Trjegul84) [reply src/main/webapp/WEB-INF/pages/inc/menu.ftl:64]: Done.

Comment (Trjegul84) [reply src/main/webapp/WEB-INF/pages/inc/menu.ftl:64]: There is no need to make changes for this files:
    src/main/webapp/WEB-INF/pages/faq.ftl
    src/main/resources/ApplicationResources_es.properties

**Review by alegnaaived — APPROVED** [review]

**Pull request merged**